iphone/metadata/en-US/release_notes.txt

```diff
@@ -3,7 +3,7 @@
 • use the map when CarPlay is connected
 • search for US zip codes and UK (GB) postcodes
 • export bookmarks and tracks in GPX file format
-• new OpenStreetMap data as of May 28
+• new OpenStreetMap data as of June 13
```

Comment:
I suggest to move the hotfix line and the new data line above everything else, e.g.
```
• fix app freeze when a bookmark in a list with a long name is selected
• new OpenStreetMap data as of June 13

• use the map when CarPlay is connected
• search for US zip codes and UK (GB) postcodes
• export bookmarks and tracks in GPX file format
...
```

Or just add the hotfix line to be the first one.

Reply:
I agree, the important fix line is missing now.